src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,25 @@
+package duke.task;
+
+/**
+ * Represents an event with a description and a time.
+ */
+public class Event extends Task {
+    
+    protected String at;
+
+    /**
+     * Creates a new Event.
+     *
+     * @param description The description of the Event.
+     * @param at String representing the time that the event occurs.
+     */
+    public Event(String description, String at) {
+        super(description);
+        this.at = at;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (at: " + at + ")";
+    }
+}
```

Comment:
Should there be a new line here?

Considering adding a new line here, as it would be flagged up during the check style.